test/documentation/version.js

```diff
@@ -0,0 +1,56 @@
+/* Jongkyu Kim (j.kim@fu-berlin.de), 2016.01.12 */
+
+var scripts = document.getElementsByTagName('script');
+var lastScript = scripts[scripts.length-1];
+var scriptName = lastScript.src;
+var split = scriptName.split("/");
+DOCUMENT_URL = split.slice(0, split.length - 2).join("/") + "/";
+
+VERSION_JSON_URL = "version.php";
+
+function changeVersion(formid)
+{
+    var frm = document.getElementById(formid);
+    window.top.location.href = DOCUMENT_URL + frm.options[frm.selectedIndex].value;
+}
+
+function addVersionSelection(arr)
+{
+    // add HTMLs
+    var version_select = document.createElement("select");
+    var version_div = document.createElement("div");
+
+    version_select.setAttribute("id","version_select");
+    version_div.setAttribute("style","vertical-align:middle; text-align:right;");
+    version_div.appendChild(document.createTextNode("Version: "));
+    version_div.appendChild(version_select);
+    document.getElementById("list_bottom_right").appendChild(version_div);
+
+    version_select.addEventListener("change", function(){changeVersion(this.id);}, false);
+
+    // current selection is..
+    cur_sel = window.location.pathname.split("/")[2];
+
+    for(i=0; i < arr.length; ++i)
+    {
+        var op = document.createElement("option");
+        op.value = arr[i];
+        op.text = arr[i];
```

Comment:
Why is op.value the same as op.text?

Reply:
value is the internal "value" that is used when the option is selected in the dropdown menu. text is the label you can see in the dropdown menu.
In this case, both are the same since we want to show, e.g. "3.0.0" and also switch the URL to "3.0.0" once we select it.